src/main/g8/build.gradle

```diff
@@ -11,7 +11,7 @@ dependencies {
     compile 'com.typesafe.akka:akka-http_2.12:$akka_http_version$'
     compile 'com.typesafe.akka:akka-stream_2.12:$akka_version$'
     testCompile 'com.typesafe.akka:akka-http-testkit_2.12:$akka_http_version$'
-    testCompile 'org.scalatest:scalatest_2.12:3.0.1'
+//    testCompile 'org.scalatest:scalatest_2.12:3.0.1' // Uncomment this dependency when using Akka HTTP 10.0.7 or prior.
```

Comment:
I'd rather remove it completely, we want to give "clean slate" and not polute new people's learning with some legacy thingy :)